src/TLuaInterpreter.cpp

```diff
@@ -10201,86 +10274,206 @@ int TLuaInterpreter::getCmdLine(lua_State* L)
 
 int TLuaInterpreter::installPackage(lua_State* L)
 {
-    string event;
+    QString packageName;
     if (!lua_isstring(L, 1)) {
-        lua_pushstring(L, "installPackage(): wrong argument type");
-        lua_error(L);
-        return 1;
+        lua_pushfstring(L, "installPackage: bad argument #1 type (package {path and/or} file name as string expected, got %s!)",
```

Comment:
> package {path and/or} file name

This is confusing English grammar... how about `package path or the file name`

Reply:
How about:

    package file name {may include a path}